README.md

```diff
@@ -66,6 +82,13 @@ And then execute:
 $ bundle install
 ```
 
+To warn on sign in, override `after_sign_in_path_for`
+```ruby
+def after_sign_in_path_for(resource)
```

Comment:
Would it be possible to move this into a controller we provide?

We would need another config flag to keep the change backwards compatible but it would avoider consumers taking a dependency on how we implement model.

Its outside the scope of this change but before releasing this we should probably add a simple code generator that install the extension and also sets our preferred config values, that way new consumers can benefit from this change without manually setting the config.

Reply:
I'm not sure. It needs to hook into the session controller to get access to the password param, so to access the pwned? status after that it would need to store that state somewhere (db, session cookie, ?) which seems more complex.

Reply:
Yeah, I was thinking we could do this through registering a controller in .add_module but seems not. Change looks good to me